packages/template-cli/templates/src/ssr/server/api/health.ts

```diff
@@ -0,0 +1,21 @@
+import { Router } from "express";
+import logger from "../core/root-logger";
+
+export default (router: Router) => {
+    router.get("/healthz", async (req, res) => {
```

Comment:
/healthz :)

Reply:
yeah trying to keep it in with kube convention